README.md

```diff
@@ -370,18 +384,6 @@ Tests both the Rust and Solidity code.
 make test
 ```
 
-### Github Actions CI
```

Comment:
Why remove that?

Reply:
We are almost not using them at all, at least I've never use it and doubt that with all the changes made in the workflows there will be many things that won't work. There is an issue from @Teebor-Choka mentioning that we need to allow external users to contribute and fix any access to secrets.

Reply:
Some things can be tested locally, like container building...